.github/workflows/tests.yml

```diff
@@ -110,6 +110,35 @@ jobs:
     - name: Check all targets
       run: cargo check --locked --all-targets --all-features
 
+  bench:
+    name: Cargo benchmarks
+    runs-on: ubuntu-22.04
+
```

Comment:
I wonder what is the motivation to add this `cargo bench` to the CI. From what I see, there is no results being stored and each run could produce different results since it partially depends on the runner's performance (which is up to GitHub to decide).

Reply:
No strong motivation really. I don't think it does us any harm to run them, and it at least checks that they still run without panicing or something, but happy to remove them if you think they're going to confuse us in the future.

Reply:
We can keep it, I'm just curious.